tests/components/homekit_controller/specific_devices/test_koogeek_ls1.py

```diff
@@ -31,18 +29,16 @@
     entry = entity_registry.async_get("light.koogeek_ls1_20833f")
     assert entry.unique_id == "homekit-AAAA011111111111-7"
 
-    helper = Helper(
-        hass, "light.koogeek_ls1_20833f", pairing, accessories[0], config_entry
-    )
+    helper = Helper(hass, "light.koogeek_ls1_20833f", pairing, accessories[0],
+                    config_entry)
     state = await helper.poll_and_get_state()
 
     # Assert that the friendly name is detected correctly
     assert state.attributes["friendly_name"] == "Koogeek-LS1-20833F"
 
     # Assert that all optional features the LS1 supports are detected
-    assert state.attributes["supported_features"] == (
-        SUPPORT_BRIGHTNESS | SUPPORT_COLOR
-    )
+    assert state.attributes["supported_features"] == (SUPPORT_BRIGHTNESS
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)